units/src/amount/signed.rs

```diff
@@ -152,6 +152,14 @@ impl SignedAmount {
         }
     }
 
+    /// Converts from a value expressing a whole number of bitcoin to a [`SignedAmount`]
+    /// in const context.
+    #[allow(clippy::missing_panics_doc)]
+    #[deprecated(since = "TBD", note = "renamed to `from_btc_i16`")]
+    pub const fn from_int_btc_const(whole_bitcoin: i16) -> SignedAmount {
```

Comment:
This also changed type so wouldn't it be better to backport the deprecation?

Reply:
Ah, you mean, backport only the deprecation but not the type change?

Reply:
To be clearer:

* Deprecate the `from_int_btc` in 0.32.x and backport the new `from_btc_i16` to 0.32.x
* Then just delete `from_int_btc` in master since its type changed (and the old type signature is untenable because of the MAX_MONEY changes)

Reply:
Yes, that's what I meant.

Reply:
Changed to just delete the old method. Separately we should backport a deprecation of it.